app/views/volunteers/show.html.slim

```diff
@@ -12,11 +12,21 @@ nav.navbar.section-navigation
         - if @volunteer.assignments.any?
           li = button_link t_title(:new, BillingExpense), new_volunteer_billing_expense_path(@volunteer)
           li = button_link t_title(:index, BillingExpense), volunteer_billing_expenses_path(@volunteer)
+        - if @volunteer.state == Volunteer::RESIGNED
+          li= button_link t('.new_certificate'), new_volunteer_certificate_path(@volunteer)
+          - if @volunteer.certificates.size == 1
+            li= button_link t('.show_certificate'), volunteer_certificate_path(@volunteer, @volunteer.certificates.first)
+          - elsif @volunteer.certificates.size > 1
+            li= button_link t('.show_certificates'), volunteer_certificates_path(@volunteer)
   ul.list-inline
     - if @volunteer.assignments.any?
-      li = button_link t_title(:new, Hour), new_volunteer_hour_url(@volunteer)
-      li = button_link t_title(:index, Hour), volunteer_hours_path(@volunteer)
-
+      li = button_link t('.report_hours'), new_volunteer_hour_url(@volunteer)
+      li = button_link t('.hour_reports'), volunteer_hours_path(@volunteer)
+    - if current_user.volunteer?
```

Comment:
Maybe reuse a policy here?

Reply:
I don't see which policy would tell me if it's a volunteer or not. This conditional is only about the text within the button.

The Volunteer should not consider editing himself. 

`t('edit_profile')` = `Profil bearbeiten` 

and the other version for the superadmins is 

`Edit Volunteer`

Reply:
The policy's will only give a true or false in case the user is a superadmin or if the user is volunteer and the record is associated to it. I'd have to create a new policy rule for this, and as this condition is only for what text is beeing displayed for the same link. So actually not a policy matter at all.

Reply:
Its a frontend and translation issue. The difference between volunteer or not is just the text in the very same button. Not a policy issue, and in my humble oppinion it shouldn't be turned into that.

Reply:
Ok makes sense!